### PR TITLE
Chore: Update test cases format and descriptions

### PR DESCRIPTION
**Pull request by jkodu**
- Updated test case format to match https://github.com/auto-wcag/auto-wcag/issues/177
- Added description to test cases of already published rules.

Closes issue:
- https://github.com/auto-wcag/auto-wcag/issues/177
- https://github.com/auto-wcag/auto-wcag/issues/229

## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**
- [x] Make sure you requesting to **pull a issue/feature/bugfix branch** (right side) to the master branch (left side).
- [x] Make sure the pull request is prefixed with `Rule:` or `Algorithm:` or `Chore:` based on work involved.

# How to Review And Approve
- Go to the “files changed” tab, there you will have the option to leave comments on different lines. 
- Once the review is completed, find the “Review changes” button in the top right, select “approve” and click “Submit review”.

### CODE REVIEW

File: _rules/SC3-1-1-html-lang-valid.md [hunk 1]
```diff
@@ -47,54 +47,102 @@ While HTML5 specification indicates that `xml:lang` attribute takes priority ove
 
 ## Passed
 
+#### Pass example 1
+
+Has valid `lang`.
```
Comment (WilcoFiers) [inline _rules/SC3-1-1-html-lang-valid.md]: This is not a grammatically correct sentence. Neither are the others in this file.

File: _rules/SC3-1-2-lang-valid.md [hunk 2]
```diff
@@ -40,6 +40,10 @@ The `lang` and `xml:lang` attributes have a [valid language subtag](#valid-langu
 
 ### Passed
 
+#### Pass example 1
+
+Valid `lang` value.
```
Comment (WilcoFiers) [inline _rules/SC3-1-2-lang-valid.md]: Better descriptions needed in this file too.

File: _rules/SC4-1-1-unique-id.md [hunk 3]
```diff
@@ -42,16 +42,22 @@ There are no major accessibility support issues known for this rule.
 
 ### Passed
 
+#### Pass example 1
+
```
Comment (WilcoFiers) [inline _rules/SC4-1-1-unique-id.md]: Lets add descriptions here too.

Comment (jkodu) [reply _rules/SC4-1-1-unique-id.md]: I have created an issue for this, as I was not the author for the same. https://github.com/auto-wcag/auto-wcag/issues/228

**Comment by jkodu** [comment]
@WilcoFiers - Please review again.

File: _rules/SC1-3-5-autocomplete-valid.md [hunk 4]
```diff
@@ -64,114 +64,177 @@ The intent of this rule is to ensure that the `autocomplete` attribute can be us
...
+#### Pass example 6
+
+Triple autocomplete terms.
+
 ```html
-<!-- Tripple autocomplete terms -->
 <input type="text" autocomplete="section-primary billing street-address />
 ```
 
+#### Pass example 7
```
Comment (WilcoFiers) [inline _rules/SC1-3-5-autocomplete-valid.md]: There's something odd going on with the code highlighter here. Is that on us or is it Github?

Comment (jkodu) [reply _rules/SC1-3-5-autocomplete-valid.md]: Seems to be a GitHub issue

**Comment by jkodu** [comment]
There is a dependency for this work to be merged, to continue on test case exporting and test runner, hence merging.

**Pull request merged**